src/main/resources/com/hermesworld/ais/galapagos/topics/service/impl/messages_de.properties

```diff
@@ -0,0 +1,30 @@
+#TopicServiceImpl class keys
+UNKNOWN_APPLICATION_ID=Unbekannte Anwendung ID.
+APPLICATION_NOT_REGISTERED_ON_ENVIRONMENT=Awendung {0} ist nicht auf der Umgebung {1} registriert.
+CANNOT_PARSE_JSON_SCHEMA=Konnte JSON-Schema nicht parsen.
+CURRENT_USER_IS_NO_OWNER_OF_APPLICATION=Der aktuelle Benutzer ist kein Besitzer der Anwendung {0}.
+FOR_COMMAND_SUBSCRIBE_TOPIC_ADD_PRODUCER=F�r den Befehl Topics abonnieren Sie das Topic, um einen neuen Producer hinzuf�gen.
```

Comment:
We use "du" in german for our users 🙂  Please update all relevant messages accordingly.